src/decorations/functions.jl

```diff
@@ -34,6 +43,11 @@ end
 in(x::T, a::DecoratedInterval) where T<:Real = in(x, interval_part(a))
 
 
+function ==(x::DecoratedInterval, y::DecoratedInterval)
+    isnai(x) && isnai(y) && return true
```

Comment:
Should two `nai` intervals be equal? Is this required by the standard?  E.g. `NaN == NaN`  is false.

Reply:
`DecoratedInterval(2, 1) == nai()`  is true according to the standard, so `NaN == NaN` must also be true.

Reply:
Ups, my mistake, I didn't see the next line. Sorry, you are right.